scripts/outer/kubectl-kubestellar-prep_for_syncer

```diff
@@ -177,15 +177,11 @@ echo "now that workspace _exists_, but is it _ready_?"
 sleep 5
 
 if ! kubectl ws "${kubectl_flags[@]}" "$mbwsname" &> /dev/null; then
-	kubectl ws .
-	echo "tried once to switch to $mbwsname"
+    echo "tried once to switch to $mbwsname"
     sleep 15
     if ! kubectl ws "${kubectl_flags[@]}" "$mbwsname" &> /dev/null; then
-	kubectl ws .
-	echo "tried twice to switch to $mbwsname"
-	sleep 15
-	kubectl ws .
-	kubectl ws "${kubectl_flags[@]}" "$mbwsname"
```

Comment:
Did you really intend to delete this line? It was deliberately put there to try one last time and err out if unsuccessful.

Reply:
If the last (could be either the 3rd, like in the current revision; or the 2nd, in the changed revision) try fails, we don't want to proceed, instead we want to be aware of the failure and explicitly exit, right?

Reply:
Yes, we want the script to fail so that the caller is aware that there is a problem. It is not necessary for the script to explicitly say `exit`; any command that fails will cause an exit because of the earlier `set -e`.
The original version of this script deliberately tries three times.